### PR TITLE
SmartGameBooster.SmartGameBooster5.2 version 5.2.0.567

### PR DESCRIPTION
**Pull request by ghost**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/27921)

### CODE REVIEW

**Comment by BajajAditya123** [comment]
Already validated and installed "C:\Users\adity>winget install --manifest "C:\Users\adity\manifests\s\SmartGameBooster\SmartGameBooster5\2\5.2.0.567"
Found Smart Game Booster 5.2 [SmartGameBooster.SmartGameBooster5.2]
This application is licensed to you by its owner.
Microsoft is not responsible for, nor does it grant any licences to, third-party packages.
Downloading https://www.pcgameboost.com/download.php
  ██████████████████████████████  57.7 MB / 57.7 MB
Successfully verified installer hash
Starting package install...
Successfully installed

**Comment by BajajAditya123** [comment]
@msftbot Can this pull request get Moderation label

File: manifests/s/SmartGameBooster/SmartGameBooster5/2/5.2.0.567/SmartGameBooster.SmartGameBooster5.2.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: SmartGameBooster.SmartGameBooster5.2
+PackageVersion: 5.2.0.567
```
Comment (OfficialEsco) [inline manifests/s/SmartGameBooster/SmartGameBooster5/2/5.2.0.567/SmartGameBooster.SmartGameBooster5.2.installer.yaml:5]:
```suggestion
PackageIdentifier: SmartGameBooster.SmartGameBooster
PackageVersion: 5.2.0
```

File: manifests/s/SmartGameBooster/SmartGameBooster5/2/5.2.0.567/SmartGameBooster.SmartGameBooster5.2.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.4.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: SmartGameBooster.SmartGameBooster5.2
+PackageVersion: 5.2.0.567
+PackageLocale: en-US
+Publisher: Smart Game Booster
+PackageName: Smart Game Booster 5.2
```
Comment (OfficialEsco) [inline manifests/s/SmartGameBooster/SmartGameBooster5/2/5.2.0.567/SmartGameBooster.SmartGameBooster5.2.locale.en-US.yaml:8]:
```suggestion
PackageIdentifier: SmartGameBooster.SmartGameBooster
PackageVersion: 5.2.0
PackageLocale: en-US
Publisher: Smart Game Booster
PackageName: Smart Game Booster
```

File: manifests/s/SmartGameBooster/SmartGameBooster5/2/5.2.0.567/SmartGameBooster.SmartGameBooster5.2.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+# Created using wingetcreate 0.4.0.3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.0.0.schema.json
+
+PackageIdentifier: SmartGameBooster.SmartGameBooster5.2
+PackageVersion: 5.2.0.567
```
Comment (OfficialEsco) [inline manifests/s/SmartGameBooster/SmartGameBooster5/2/5.2.0.567/SmartGameBooster.SmartGameBooster5.2.yaml:5]:
```suggestion
PackageIdentifier: SmartGameBooster.SmartGameBooster
PackageVersion: 5.2.0
```

**Review by OfficialEsco — CHANGES_REQUESTED** [review]

File: manifests/s/SmartGameBooster/SmartGameBooster5/2/5.2.0.567/SmartGameBooster.SmartGameBooster5.2.yaml (hunk 3 above)
Comment (BajajAditya123) [reply manifests/s/SmartGameBooster/SmartGameBooster5/2/5.2.0.567/SmartGameBooster.SmartGameBooster5.2.yaml:5]: I have done these changes in #27970
This should work

**Comment by BajajAditya123** [comment]
I have done these changes in #27970

**Pull request closed without merging**